SignalServiceKit/src/Contacts/CDSSigningCertificate.h

```diff
@@ -2,11 +2,28 @@
 //  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
 //
 
+#import "OWSError.h"
+
 NS_ASSUME_NONNULL_BEGIN
 
+typedef NS_ENUM(NSUInteger, CDSSigningCertificateErrorCode) {
```

Comment:
This is a diversion from our usual approach of piling everything into OWSError, but I'm not sure there's much value in having everything piled into OWSError - the downside of doing it that way is I think error codes get - reused for different things, and having smaller enums defined within the context of their usage gives a better hint as to what they mean.

Instead, I'm proposing that we  (roughly) use the classname (or related class name in the case of small internal classes that produce errors) as the domain and defining error codes scoped to that.

Reply:
That seems reasonable.